pkg/cmd/roachtest/tests/online_restore.go

```diff
@@ -109,6 +148,7 @@ func registerOnlineRestore(r registry.Registry) {
 							if runOnline {
 								opts = "WITH EXPERIMENTAL DEFERRED COPY"
 							}
+							restoreStartTime = timeutil.Now()
```

Comment:
do you need to send this on a chan, and receive it below? otherwise there's no coordination is there?

Reply:
I don't think it's necessary, as we use it after waiting for these goroutines to return.

Reply:
👍 aha, GitHub chopped it off one line after the `m.Wait`. carry on.